data/links.json

```diff
@@ -212,6 +212,11 @@
         "href": "https://starrs.csh.rit.edu",
         "icon": "material:signal-wifi-4-bar-lock"
       },
+      {
+        "name": "Proxstar",
```

Comment:
I think the plan is to just refer to it as vm.csh/VM Management. So, just change this to “VM Management”.